.github/workflows/release.yaml

```diff
@@ -0,0 +1,25 @@
+# build and release new docs as a release asset
+name: Release Docs
+on:
+  release:
+    types: [created]
+jobs:
+  doc:
+    name: Bundle and Release Docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
+        with:
+          python-version: "3.13"
+      - name: Install pipx
+        run: python -m pip install pipx
+      - name: Build the Doc Bundle
+        run: |
+          pipx run ./support/build-doc-bundle.py -o doc_bundle.tar.gz
+      # upload as a release asset
+      - name: Upload Doc Bundle
+        env:
+          GH_TOKEN: ${{ github.token }}
+          GH_REPO: ${{ github.repository }}
```

Comment:
I don't think that `GH_REPO` is needed; I assume that it's picked up automatically based on my experience elsewhere.

```suggestion
```

Also, I strongly suspect that this needs explicit write permissions so the GitHub token has permissions to write assets to the release.

```yaml
permissions:
  contents: "write"
```

Reply:
Hm. I wonder if we have some repo-settings for globus-cli that allow the similar workflow there to run without the explicit permissions? I agree now that you've raised it that we should need it here.

Reply:
It might be! However, one of the benefits to listing the permissions explicitly is that it _forces the token to only have those permissions_ in the context that it's running in.

(I'm actually working to get better at explicitly listing read-only permissions in workflows, so that it's impossible to inherit something unexpectedly.)